Modules/_datetimemodule.c

```diff
@@ -2832,19 +2832,26 @@ date_today(PyObject *cls, PyObject *dummy)
     return result;
 }
 
-/* Return new date from given timestamp (Python timestamp -- a double). */
+/*[clinic input]
+@classmethod
+datetime.date.fromtimestamp
+
+    timestamp: 'O'
```

Comment:
`timestamp: object`

Reply:
Thanks. Didn't get to that point in the clinic documentation.